### PR TITLE
[P4-1456] Show incoming/outgoing moves on dashboard

### PR DESCRIPTION
**Pull request by teneightfive**
### What changed

This set of changes adds the outgoing and incoming panels to the dashboard. It re-uses the existing patterns for filters to display the results for the default time period.

### Why did it change

The incoming section was recently added to the app and we need a way to show a summary of those sections on the dashboard so this change adds both outgoing and incoming to the dashboard in the same style as the other sections.

### Issue tracking

- [P4-1456](https://dsdmoj.atlassian.net/browse/P4-1456)

## Screenshots

| Before | After |
| ------ | ----- |
| <kbd></kbd> | <kbd></kbd> |
| <kbd></kbd> | <kbd></kbd> |
| N/A | <kbd></kbd> |

#### Automated testing

- [ ] Added unit tests to cover changes
- [ ] Added end-to-end tests to cover any journey changes

#### Manual testing

Has been tested in the following browsers:

- [x] Chrome
- [x] Firefox
- [ ] Edge
- [ ] Internet Explorer

### Environment variables

- [x] No environment variables were added or changed

### Other considerations

Commit messages with a `fix` or `feat` type are automatically used to generate the [changelog](ministryofjustice/hmpps-book-secure-move-frontend/blob/master/CHANGELOG.md) and
[GitHub release notes](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/releases) during the release task. Please make sure they will read well on their own in a
summary of changes and that the commit body gives a more detailed description of those changes if necessary.

- [x] No new [Personally Identifiable Information (PII)](https://support.google.com/analytics/answer/6366371?hl=en) is being sent via analytics
- [x] Update [README](ministryofjustice/hmpps-book-secure-move-frontend/blob/master/README.md) with any new instructions or tasks

### CODE REVIEW

File: app/home/middleware.js [hunk 1]
```diff
@@ -1,37 +1,11 @@
...
-    fromLocationId: get(req.session, 'currentLocation.id'),
-  }
-
-  req.body.allocations = {
-    ...req.body.allocations,
-    moveDate: [startDate, endDate],
-    fromLocationId: get(req.session, 'currentLocation.id'),
-  }
+function overrideLocationId(req, res, next) {
+  req.params.locationId = get(req.session, 'currentLocation.id')
```
Comment (solidgoldpig) [inline app/home/middleware.js:4]: Test?

File: common/helpers/date-utils.js [hunk 2]
```diff
@@ -13,6 +13,28 @@ const { DATE_FORMATS } = require('../../config/index')
 
 module.exports = {
   dateFormat: DATE_FORMATS.URL_PARAM,
+  getCurrentWeekAsRange: () => {
```
Comment (solidgoldpig) [inline common/helpers/date-utils.js:16]: Tests?

File: common/helpers/date-utils.js [hunk 3]
```diff
@@ -13,6 +13,28 @@ const { DATE_FORMATS } = require('../../config/index')
...
+    const endDate = format(
+      endOfWeek(today, {
+        weekStartsOn: DATE_FORMATS.WEEK_STARTS_ON,
+      }),
+      DATE_FORMATS.URL_PARAM
+    )
+
+    return [startDate, endDate]
+  },
+  getCurrentDayAsRange: () => {
```
Comment (solidgoldpig) [inline common/helpers/date-utils.js:33]: Tests?

File: common/helpers/date-utils.js (hunk 2 above)
Comment (teneightfive) [reply common/helpers/date-utils.js:16]: Coming ;) It's still in draft but wanted to get some early eyes on the approach to make sure it made sense.

File: app/home/middleware.js (hunk 1 above)
Comment (teneightfive) [reply app/home/middleware.js:4]: Draft ;)

**Pull request merged**